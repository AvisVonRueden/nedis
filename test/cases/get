*3
$3
SET
$5
hello
$5
world
--------
*2
$3
GET
$5
hello
-------
*2
$3
DEL
$5
hello
-------
*4
$4
HSET
$6
user:0
$4
name
$2
tj
-------
*2
$3
GET
$6
user:0
