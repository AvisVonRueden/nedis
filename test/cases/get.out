+OK
$5
world
